Spokestack/TextToSpeechInput.swift

```diff
@@ -8,46 +8,47 @@
 
 import Foundation
 
+/// Designate the format the input is provided in.
+@objc public enum TTSInputFormat: Int {
+    @available(*, unavailable, message: "Text-only input is not currently supported. Encase your text with <speak>text</speak> for SSML compatibility.")
+    /// Plain text
+    case text
+    /// Speech Synthesis Markup Language
+    /// see https://spokestack.io/docs/Concepts/tts for the supported SSML subset.
+    case ssml
+}
+
+@objc public enum TTSInputVoice: Int {
+    case demoMale
+}
+
 /// Input parameters for speech synthesis. Parameters are considered transient and may change each time `synthesize` is called.
 /// - SeeAlso: `TextToSpeech.synthesize`
 @objc public class TextToSpeechInput: NSObject {
     
     /// Initializer for a new TextToSpeechInput instance.
-    @objc public override init() {
-        super.init()
-    }
-    
-    /// Convenience initializer for a new TextToSpeechInput instance.
-    /// - Parameter input: The text input to the speech synthesizer.
-    @objc public init(_ input: String) {
-        self.input = input
-        super.init()
-    }
-
-    /// Convenience initializer for a new TextToSpeechInput instance.
-    /// - Parameter input: The text input to the speech synthesizer.
-    /// - Parameter voice: The synthetic voice used to generate speech.
-    /// - Parameter inputFormat: The formatting of the input.
-    @objc public init(_ input:String, inputFormat: TTSInputFormat) {
-        self.input = input
-        self.inputFormat = inputFormat
-    }
-    
-    /// Convenience initializer for a new TextToSpeechInput instance.
     /// - Parameter input: The text input to the speech synthesizer.
     /// - Parameter voice: The synthetic voice used to generate speech.
     /// - Parameter inputFormat: The formatting of the input.
-    @objc public init(_ input:String, voice: String, inputFormat: TTSInputFormat) {
+    /// - Parameter id: A unique identifier for this input request.
+    @objc public init(_ input:String = "<speak>Here I am, a brain the size of a planet.</speak>",
```

Comment:
A default value for the text to synthesize seems unnecessary or maybe even counterproductive—or is there an xcode-related reason for this?

Reply:
Because this has to be a class for `@objc` compatibility, it must either be initialized or optional. in cases like this, setting a default value allows for both a simpler implementation and a no-configuration-necessary client use case. In scenarios where the client does not directly use a property, I use optionals.

Reply:
So just including it in the initializer without a default value doesn't count as it being initialized?

Reply:
well, i am surprised to find out that class properties indeed do not require default values if they are initialized in the only provided constructor.